### PR TITLE
Adding modifications of gadgets and user analytics dashboard

### PR DESCRIPTION
**Pull request by dewmini**

### CODE REVIEW

File: modules/extensions/apps/portal/store/carbon.super/fs/gadget/user_level_task_instance_count_vs_task_id/js/actions.js [hunk 1]
```diff
@@ -44,6 +45,7 @@ function drawGraph() {
 
                     responseStr = responseStr.slice(0, -1);
                     var jsonArrObj = JSON.parse('[' + responseStr + ']');
+                    //console.log("hhhhhhhhhhhhhhhhhhhhhhhhhhh"+jsonObj);
```
Comment (chamindias) [inline modules/extensions/apps/portal/store/carbon.super/fs/gadget/user_level_task_instance_count_vs_task_id/js/actions.js]: Please remove this : console.log("hhhhhhhhhhhhhhhhhhhhhhhhhhh"+jsonObj);

File: modules/extensions/apps/portal/store/carbon.super/fs/gadget/avg_execution_time_vs_task_id/index.xml [hunk 2]
```diff
@@ -84,7 +84,7 @@
                             <option value="100">100</option>
                         </select>
                     </div>
-                    <button class="btn btn-default btn-info" onclick="drawGraph()">Apply Filter</button>
+                    <button class="btn btn-primary" onclick="drawGraph()">Apply Filter</button>
```
Comment (anuruddhal) [inline modules/extensions/apps/portal/store/carbon.super/fs/gadget/avg_execution_time_vs_task_id/index.xml]: Does this support internationalization?
{{i18n Apply Filter}}

File: modules/extensions/apps/portal/store/carbon.super/fs/gadget/user_level_task_instance_count_vs_task_id/js/actions.js (hunk 1 above)
Comment (dewmini) [inline modules/extensions/apps/portal/store/carbon.super/fs/gadget/user_level_task_instance_count_vs_task_id/js/actions.js]: Fixed in 73245989e1153871733cabee82f55eea843f17b7

File: modules/extensions/apps/portal/store/carbon.super/fs/gadget/avg_execution_time_vs_task_id/index.xml (hunk 2 above)
Comment (dewmini) [inline modules/extensions/apps/portal/store/carbon.super/fs/gadget/avg_execution_time_vs_task_id/index.xml]: This does not support internationalization.

File: modules/extensions/apps/portal/store/carbon.super/fs/gadget/total_involved_instance_count_vs_process_id/js/actions.js [hunk 3]
```diff
@@ -96,12 +97,34 @@ function isInteger(param) {
...
+    user = getUrlVars()["user"];
+
+    $.getJSON("/portal/store/carbon.super/fs/gadget/total_involved_instance_count_vs_process_id/js/meta-data-totalInstancesVsProcessID.json.js", function (result) {
+        $.each(result, function (i, field) {
+            jsonObj.push(field);
+            if (user) {
+                var el = document.createElement("option");
+                el.textContent = user;
+                el.value = user;
+                $(dropdownElementID).append(el);
```
Comment (dakshika) [inline modules/extensions/apps/portal/store/carbon.super/fs/gadget/total_involved_instance_count_vs_process_id/js/actions.js]: cache the variable

File: modules/extensions/apps/portal/store/carbon.super/fs/gadget/total_involved_time_vs_process_id/js/actions.js [hunk 4]
```diff
@@ -96,12 +97,33 @@ function isInteger(param) {
...
+    user = getUrlVars()["user"];
 
+    $.getJSON("/portal/store/carbon.super/fs/gadget/total_involved_time_vs_process_id/js/meta-data-totalTimeVsProcessID.json.js", function (result) {
+        $.each(result, function (i, field) {
+            jsonObj.push(field);
+            if (user) {
+                var el = document.createElement("option");
+                el.textContent = user;
+                el.value = user;
+                $(dropdownElementID).append(el);
```
Comment (dakshika) [inline modules/extensions/apps/portal/store/carbon.super/fs/gadget/total_involved_time_vs_process_id/js/actions.js]: cache the variable

**Pull request merged**